open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/api/SearchIntegratorAPI.java

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.integrationservices.search.api;
+
+import org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.UserNotAuthorizedException;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.ConnectorType;
+
+
+/**
+ * LineageIntegratorAPI is the interface to validate that a connector is suitable to run in the Lineage Integrator OMIS.
```

Comment:
Should LineageIntegratorAPI be SearchIntegratorAPI

Reply:
updated